yaml/simulator.yaml

```diff
@@ -108,23 +108,26 @@
     cov_opts: >
       -coverage -ucdb <out>/cov.ucdb
 
-- tool: riviera
-  compile:
-    cmd:
-      - "vlib <out>/work"
-      - "vlog -work <out>/work
-        -err VCP2694 W1
-        -uvmver 1.2
-        +define+UVM_REGEX_NO_DPI
-        +incdir+<setting>
-        +incdir+<user_extension>
-        -f <cwd>/files.f
-        <cmp_opts>"
-  sim:
-    cmd: >
-      vsim -c <sim_opts> -sv_seed <seed> <cov_opts> -do <cwd>/riviera_sim.tcl
-    cov_opts: >
-      -acdb_file <out>/cov.acdb
+#RIVIERA REMOVED BECAUSE RANDOMIZATION ISSUE
+#   ADDITIONALLY, 10X SLOWER THAN VCS
```

Comment:
Recommend against making vendor-specific comments.

Reply:
Fair enough, I suppose we should try to resubmit the pull-request where we remove things that are specific to our design. I forgot we had an open pull request, so perhaps we can reorganize things a bit and move the Gaisler specific changes to another branch and then keep new extensions plus bugfixes in the branch we try to merge from.

For this particular comment we can take another look to see if we can get Riviera working, we are having issues with Riviera only randomizing the operands once per instruction, this behavior is not something we reproduce in VCS.

Reply:
Better yet, let's get Aldec looking into this.   I have some contacts there.  If you agree, I'll reach out to them about this issue.